client/state/jetpack-connect/selectors.js

```diff
@@ -1,16 +1,49 @@
 /**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
  * Internal dependencies
  */
 import { getSiteByUrl } from 'state/sites/selectors';
 
 const JETPACK_CONNECT_TTL = 60 * 60 * 1000; // an hour
 
+const getConnectingSite = ( state ) => {
+	return get( state, [ 'jetpackConnect', 'jetpackConnectSite' ] );
```

Comment:
I may be wrong here, but I think we don't need to use `get`, since state.jetpackConnect is going to always be there